### PR TITLE
Add asar support. Closes GH-2

### PR DESCRIPTION
**Pull request by shama**
`electron-packager . MyApp --asar`

Thanks!

### CODE REVIEW

File: linux.js [hunk 1]
```diff
@@ -43,6 +45,11 @@ module.exports = {
     function renameElectronBinary () {
       fs.rename(originalBinary, finalBinary, function electronRenamed (err) {
         if (err) return cb(err)
+        if (opts.asar) {
+          asarApp(cb)
+        } else {
+          cb()
```
Comment (malept) [inline linux.js]: Is this necessary?

Comment (shama) [inline linux.js]: I thought so as it didn't seem to call `cb()` which for the cli probably doesn't matter but if someone was using it as a lib they would want to know when the `cb()` was called.

**Comment by mafintosh** [comment]
@malept like @shama said we need to call the callback for module support

**Comment by mafintosh** [comment]
Hmm it seems i don't have publish rights. I already bumped the version so as soon as @maxogden wakes up I can publish this to npm.

**Comment by maxogden** [comment]
published

**Pull request merged**